echoJapanese.php

```diff
@@ -5,7 +5,16 @@
 function execute() {
     $return = "";
 
-    $return .= "あ";
+    $return .= returnあ();
+    $return .= returnか()
 
     return $return;
 }
+
+function returnあ() {
+    return "あ";
+}
+
+function returnか() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->